README.md

```diff
@@ -0,0 +1,13 @@
+# General codecs
+
+This repository contains build scripts for general codecs:
++ [fix](codec-fix/README.md)
++ [itch](codec-itch/README.md)
++ [fast](codec-fast/README.md)
++ [ntg (native)](codec-ntg/README.md)
+
+The available parameters for each codec you can find in README files in codecs' folders.
+
+If you need to modify some parameters, please, change only parameters you need.
```

Comment:
If you need to modify some parameters, please, change only **the** parameters **that** you **will** need.